zipline/data/minute_bars.py

```diff
@@ -269,12 +270,14 @@ def __init__(
         start_session,
         end_session,
         minutes_per_day,
+        version,
     ):
         self.calendar = calendar
         self.start_session = start_session
         self.end_session = end_session
         self.ohlc_ratio = ohlc_ratio
         self.minutes_per_day = minutes_per_day
+        self.version = version
```

Comment:
So when we write, we'll always be using the most recent version, i.e. `FORMAT_VERSION` (https://github.com/quantopian/zipline/pull/1425/files#diff-ad0e2225f40bfedeadd927fb0d674d7bR461).

When reading, the version could potentially be older, so we look at the version to decide how to interpret everything else.